ql/src/test/queries/clientpositive/sysdb_schq.q

```diff
@@ -0,0 +1,38 @@
+--! qt:scheduledqueryservice
+--! qt:dataset:src
+
+source ../../metastore/scripts/upgrade/hive/hive-schema-4.0.0.hive.sql;
+
+use sys;
+
+create scheduled query asd cron '* * * * * ? *' defined as select 1;
+
+!sleep 5;
```

Comment:
This option was not existing before, right? This is very useful, we should document it!